src/Admin/AbstractAdmin.php

```diff
@@ -2208,22 +2097,22 @@ public function showIn($context)
         }
     }
 
-    public function createObjectSecurity($object): void
+    public function createObjectSecurity(object $object): void
     {
         $this->getSecurityHandler()->createObjectSecurity($this, $object);
     }
 
-    public function setSecurityHandler(SecurityHandlerInterface $securityHandler): void
+    public function setSecurityHandler(?SecurityHandlerInterface $securityHandler): void
     {
         $this->securityHandler = $securityHandler;
     }
 
-    public function getSecurityHandler()
+    public function getSecurityHandler(): ?SecurityHandlerInterface
     {
         return $this->securityHandler;
     }
 
-    public function isGranted($name, $object = null)
+    public function isGranted($name, ?object $object = null): bool
```

Comment:
```suggestion
    public function isGranted(string $name, ?object $object = null): bool
```

Reply:
This argument is provided as argument 2 for `SecurityHandlerInterface::isGranted()`:
https://github.com/sonata-project/SonataAdminBundle/blob/acdfcc0ddee8e8de05b51a4ab22a562b7ea5036f/src/Security/Handler/SecurityHandlerInterface.php#L23-L29